changelogs/unreleased/gh-10636-json-schema.md

```diff
@@ -3,3 +3,9 @@
  * Added `<schema object>:jsonschema()` method to
    `experimental.config.utils.schema` for generating a JSON schema
    representation as a Lua table (gh-10636).
+
+## feature/config
```

Comment:
The `tools/gen-release-notes` expects one section per file.

In fact, both features relates to the `## feature/config` section (because the schema utility module is in the config's namespace).

It is better to split it to two files. But it is OK to have one file with two items:

```
## feature/config

* foo (gh-xxxxx).
* bar (gh-xxxxx).
```

See https://github.com/tarantool/tarantool/blob/master/doc/changelogs.md for details.

Reply:
Splited to two files.